paasta_tools/cli/cmds/validate.py

```diff
@@ -149,6 +164,31 @@ def get_service_path(service, soa_dir):
             return None
 
 
+def path_to_soa_dir_service(service_path):
+    soa_dir = os.path.dirname(service_path)
+    service = os.path.basename(service_path)
+
+    return soa_dir, service
+
+
+def validate_chronos(service_path):
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    instance_type = 'chronos'
+
+    for cluster in list_clusters(service, soa_dir, instance_type):
+        for instance in list_all_instances_for_service(service, instance_type, soa_dir):
```

Comment:
You have to pass in the cluster here, otherwise you will list instances for clusters you are not interested in. Please use kwargs